Gemfile

```diff
@@ -7,7 +7,7 @@ end
 
 gemspec
 
-# We need a newish Rake since Active Job sets its test tasks' descriptions.
+# We need a newish Rake since Active Job sets its test tasks descriptions.
```

Comment:
Can we change this sentence to `We need a newish Rake since Active Job sets descriptions for its test tasks.`